assets/sass/_chooser.scss

```diff
@@ -116,6 +116,7 @@ $tab-font-size: 14px !default;
 .mdl-tabs__tab.language-tab,
 .mdl-tabs__tab.os-tab,
 .mdl-tabs__tab.cloud-tab,
+.mdl-tabs__tab.input-kind-tab,
```

Comment:
Did you find that you needed these? I'd have guessed since you're using choosers, you wouldn't need to bother with the older chooser stuff.

Reply:
I didn't. I wasn't sure how choosers actually worked under the hood until getting through it all, so I started by copying what we (recently) did with k8s-language. I had a suspicion this wasn't required ... I'll get rid of it.